microsoft-edge/extensions-chromium/publish/add-ons-curation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Curation and review process for extensions at Microsoft Edge Add-ons
+description: How an extension that's submitted at Microsoft Edge Add-ons is curated and reviewed by the Microsoft Edge extensions team.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: extensions
+ms.date: 10/13/2023
+---
+# Curation and review process for extensions at Microsoft Edge Add-ons
+
+The Microsoft Edge extensions team curates the experience at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) so that users can find all the items to customize their browser in one place, such as themes, extensions, and apps.
+
+Available _items_ to customize the user's browser includes themes, extensions, and apps.
+
+Before installing an item from [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/), users can learn more about the functionality, data permissions, and publisher by viewing the item's description and listing.
+
+
+<!-- ====================================================================== -->
+## How items are organized at Microsoft Edge Add-ons
+
+Items at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/), such as themes, extensions, and apps, are organized to help users discover the ones that are relevant to their interests.
+
+Users can discover extensions they're interested in at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) in several ways, including:
+* The home page.
+* The Search bar.
+* Curated collections of extensions at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).
+* Categories of extensions at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).
+
+The following factors are used to organize items:
+* **Quality:** Extensions that are featured at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) are selected based on their eligibility criteria.
+* **Relevancy:** Ranking takes into consideration factors such as the item name, description relevancy, popularity, and user experience.
+* **Popularity:** The number of ratings and the average rating are used for prioritizing items.
+
+
+<!-- ====================================================================== -->
+## Security review
+
+The quality and compliance of the item that's displayed at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) is verified per the guidelines that are set in the developer policy.
+
+Extensions that violate the policy guidelines are not allowed at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).  There is a standard review process for each new item submission and new versions of items, per the developer policies.
+
+See also:
+* [Using Content Security Policy (CSP) to control which resources can be run](../developer-guide/csp.md)
```

Comment:
This isn't the right link to use here. Content Security Policy (CSP) is an API for web developers. We need to point to our extension developer policy instead. I think that's this one: https://learn.microsoft.com/en-us/legal/microsoft-edge/extensions/developer-policies

Reply:
```
See also:
* [Developer policies for the Microsoft Edge Add-ons store](/legal/microsoft-edge/extensions/developer-policies)
```